tests/system/test_integration_acm.py

```diff
@@ -0,0 +1,247 @@
+import time
+import unittest
+from urlparse import urljoin
+import uuid
+
+import requests
+
+from apmserver import ElasticTest
+from beat.beat import INTEGRATION_TESTS
+
+
+class AgentConfigurationIntegrationTest(ElasticTest):
+    config_overrides = {
+        "logging_json": "true",
+        "kibana_enabled": "true",
+        "acm_cache_expiration": "1ms",  # no cache
+    }
+
+    def create_service_config(self, settings, name, env=None, _id="new"):
```

Comment:
This function has me thinking about a subcommand for apm-server that manipulates agent configs